### ISSUE TITLE
'Show Damage on XP Drop' innaccurate vs certain bosses

### ISSUE THREAD

**Issue by datashe12** [open-osrs/runelite#2305]
The 'Show Damage on XP Drop' feature is inaccurate against zulrah and vorkath and im assuming other bosses which I haven't tested. The number shown in the xp drop is consistently ~84% of the actual hit. This happens on every hit without fail. 

https://i.imgur.com/bYFy0zD.png
https://i.imgur.com/VPg5ZeT.png
https://i.imgur.com/MUlEP9T.png

**Comment by AzeroRS** [comment]
I've been checking this issue out for a while now and I've figured out that Vorkath doesn't add any bonus health xp, the normal monster hitpoint ratio is 1.33 and vorkath's is too, if you hit a 100, you get 133 hp xp.

So what's happening here is that the plugin is calculating stats based on the monsters combat levels, defense and strength bonus, taking that and dividing your (hitpoints XP on hit/hitpoints_ratio) / expectedXPModifier. (which is 1.2 for vorkath) when there's a flat 1.0 XP Modifier for vorkath.

Same for Zulrah apparently, seems like instanced bosses do not reward bonus hp xp and that's why the plugin is calculating wrong.

Checked other instanced bosses and skotizo adds bonus HP xp. 

Grotesque Guardians do not give bonus xp. 
Kraken neither.
KBD does.
Corporal Beast does.
